src/intersection-observer.ts

```diff
@@ -140,13 +144,15 @@ export class SpanielIntersectionObserver implements IntersectionObserver {
   }
 }
 
-function addRatio(entryInit: SpanielIntersectionObserverEntryInit): IntersectionObserverEntry {
-  const { time, rootBounds, boundingClientRect, intersectionRect, target } = entryInit;
+function addRatio(entryInit: SpanielIntersectionObserverEntryInit): InternalIntersectionObserverEntry {
+  const { unixTime, highResTime, rootBounds, boundingClientRect, intersectionRect, target } = entryInit;
   const boundingArea = boundingClientRect.height * boundingClientRect.width;
   const intersectionRatio = boundingArea > 0 ? (intersectionRect.width * intersectionRect.height) / boundingArea : 0;
 
   return {
-    time,
+    time: unixTime,
+    unixTime,
```

Comment:
Can we only keep 1? It looks like other places `time` is also just aliasing `unixTime`. It is difficult to keep track of the references.

Reply:
removed